specs/notation-config.md

```diff
@@ -0,0 +1,59 @@
+# Notation Configuration
+
+To enable persisted configuration, simplifying the execution of the `notation` cli, the following configuration file will be available
+
+> Note: there will be a policy based configuration that will come at a later point.
+
+## Location
+
+The default location and file will be stored at: `~/.notary/notation-config.json`. The `notation` cli and libraries will support alternate locations through a `config-location` parameters.
```

Comment:
It seems for Unix / Linux only. How about the locations on Windows or macOS?

Reply:
Sure, what should we articulate those?

Reply:
I've added a TODO section so we can merge and come back to address